pace.coffee

```diff
@@ -202,7 +202,7 @@ class Bar
       @el = document.createElement 'div'
       @el.className = "pace pace-active"
 
-      document.body.className = document.body.className.replace /pace-done/g, ''
+      document.body.className = document.body.className.replace / pace-done/g, ''
```

Comment:
Won't this only work if the class is not the first class on the body? Maybe you mean `/(^| )pace-done/g`?